api/v1alpha1/workflow_webhook.go

```diff
@@ -181,25 +190,27 @@ func (w *Workflow) ValidateDelete() error {
 	return nil
 }
 
-func validateWorkflowImmutable(a *Workflow, b *Workflow) error {
-	if a.Spec.WLMID != b.Spec.WLMID {
-		return fmt.Errorf("cannot change immutable field WLMID")
+func validateWorkflowImmutable(wfNew *Workflow, wfOld *Workflow) error {
```

Comment:
name change from a/b is good, but oldWorkflow/newWorkflow is used elsewhere in this file. Try to stay consistent.